packages/backend/tests/lib/rankHelpers.test.ts

```diff
@@ -0,0 +1,26 @@
+import { PlayerRank_Enum } from '../../src/lib/autogen/hasura-sdk';
+import { computeRank, RANKED_CAP } from '../../src/lib/rankHelpers';
+
+describe('RANKED_CAP', () => {
+  it('should equal 56', () => {
+    expect(RANKED_CAP).toBe(56);
+  });
+});
+
+describe('computeRank', () => {
+  it('index 6 should be diamond', () => {
+    expect(computeRank(6)).toBe(PlayerRank_Enum.Diamond)
+  });
+  it('index 7 should be platinum', () => {
+    expect(computeRank(7)).toBe(PlayerRank_Enum.Platinum)
+  });
+  it('index 21 should be silver', () => {
```

Comment:
Should also test for Gold no?

Reply:
yeah, i didn't bother because I figured I hit any off-by-one errors already. and that's really what these are testing.

Reply:
Good to add it anyway for regressions testing

Reply:
done and done